### PR TITLE
feat: add docs for partnership with CrabNebula

### PR DESCRIPTION
**Pull request by lucasfernog-crabnebula**
We've been discussing about including more information about Tauri partners in the Tauri documentation, including their offerings for the community. This PR includes basic information about CrabNebula free DevTools website for instrumenting apps and the CrabNebula Cloud platform for distributing Tauri apps.

This PR targets the v1 documentation. A separate PR for v2 will be opened shortly after this gets merged, including any proposed changes.

### CODE REVIEW

File: docs/guides/debugging/crabnebula-devtools.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+}
+```
+
+:::note
+In this case we only initialize the devtools plugin for debug applications, which is recommended.
+:::
+
+For more information, see the [CrabNebula DevTools] documentation.
+
+[crabnebula devtools]: https://docs.crabnebula.dev/devtools/get-started/
```
Comment (dreyfus92) [inline docs/guides/debugging/crabnebula-devtools.md]:
```suggestion
[CrabNebula DevTools]: https://docs.crabnebula.dev/devtools/get-started/
```

**Review by dreyfus92 — COMMENTED** [review]
Hey Lucas this looks good to me, I just left a tiny suggestion 😁

File: docs/guides/debugging/crabnebula-devtools.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+@ -0,0 +1,37 @@
```
Comment (vasfvitor) [inline docs/guides/debugging/crabnebula-devtools.md]:
```suggestion
```

**Review by vasfvitor — CHANGES_REQUESTED** [review]

File: docs/guides/debugging/crabnebula-devtools.md (hunk 1 above)
Comment (FabianLars) [reply docs/guides/debugging/crabnebula-devtools.md]: iirc vscode will autoformat this to lowercase, at least it always did for me in the v1 docs

Comment (dreyfus92) [reply docs/guides/debugging/crabnebula-devtools.md]: > iirc vscode will autoformat this to lowercase, at least it always did for me in the v1 docs

Other links were formatted the same as declared. That's why I thought this was required, if that isn't the case all good, we can omit it 😁

Comment (lucasfernog-crabnebula) [reply docs/guides/debugging/crabnebula-devtools.md]: I was just following the example of the other vscode link - but you're right in other places it's listed as defined

**Review by dreyfus92 — APPROVED** [review]

**Review by vasfvitor — APPROVED** [review]

File: docs/guides/debugging/crabnebula-devtools.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# Debugging in CrabNebula DevTools
+
+CrabNebula provides a free DevTools application for Tauri as part of its partnership with the Tauri project.
```
Comment (robertfrank-crabnebula) [inline docs/guides/debugging/crabnebula-devtools.md]:
```suggestion
[CrabNebula](https://crabnebula.dev) provides a free [DevTools](https://crabnebula.dev/devtools/) application for Tauri as part of its partnership with the Tauri project.
```

File: docs/guides/distribution/distributing-with-crabnebula-cloud.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# Distributing with CrabNebula Cloud
+
+CrabNebula is an official Tauri partner providing services and tooling for Tauri applications.
```
Comment (robertfrank-crabnebula) [inline docs/guides/distribution/distributing-with-crabnebula-cloud.md]:
```suggestion
[CrabNebula](https://crabnebula.dev) is an official Tauri partner providing services and tooling for Tauri applications.
```

File: docs/guides/distribution/distributing-with-crabnebula-cloud.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+The CrabNebula Cloud is a platform for application distribution that seamlessly integrates with the Tauri updater.
```
Comment (robertfrank-crabnebula) [inline docs/guides/distribution/distributing-with-crabnebula-cloud.md]:
```suggestion
The [CrabNebula Cloud](https://crabnebula.dev/cloud/) is a platform for application distribution that seamlessly integrates with the Tauri updater.
```

File: docs/guides/debugging/crabnebula-devtools.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+This application allows you to instrument your Tauri app by capturing its embedded assets, Tauri configuration file, logs and spans and providing a web frontend to seamlessly visualize data in real time.
+
+With the CrabNebula DevTools you can inspect your app's log events (including logs from dependencies), track down the performance of your command calls and overall Tauri API usage, with special interface for Tauri events and commands, including payload, responses and inner logs and execution spans.
```
Comment (robertfrank-crabnebula) [inline docs/guides/debugging/crabnebula-devtools.md]:
```suggestion
With the CrabNebula DevTools you can inspect your app's log events (including logs from dependencies), track down the performance of your command calls and overall Tauri API usage, with a special interface for Tauri events and commands, including payload, responses and inner logs and execution spans.
```

**Review by robertfrank-crabnebula — COMMENTED** [review]
Added links to the CrabNebula homepage and the respective product pages.

File: docs/guides/debugging/crabnebula-devtools.md (hunk 3 above)
Comment (FabianLars) [reply docs/guides/debugging/crabnebula-devtools.md]: hmm, thinking back to the partners overview/intro page we want to have, do you think it makes sense to swap out the links once that's live?

Comment (lucasfernog-crabnebula) [reply docs/guides/debugging/crabnebula-devtools.md]: I like that idea

**Review by simonhyll — APPROVED** [review]

**Pull request merged**